drivers/overlay/ov_network.go

```diff
@@ -18,24 +20,24 @@ import (
 type networkTable map[string]*network
 
 type network struct {
-	id          string
-	vni         uint32
-	dbIndex     uint64
-	dbExists    bool
-	sbox        osl.Sandbox
-	endpoints   endpointTable
-	ipAllocator *ipallocator.IPAllocator
-	gw          net.IP
-	vxlanName   string
-	driver      *driver
-	joinCnt     int
-	once        *sync.Once
-	initEpoch   int
-	initErr     error
+	id        string
+	vni       uint32
+	dbIndex   uint64
+	dbExists  bool
+	sbox      osl.Sandbox
+	endpoints endpointTable
+	vxlanName string
+	driver    *driver
+	joinCnt   int
+	once      *sync.Once
+	initEpoch int
+	initErr   error
+	subnets   []*net.IPNet
```

Comment:
@mrjana For multiple subnet handling we need more fields in the subnet structure. As part of my changes I am defining the subnet structure and moved the gw field into it.